_episodes/13-scheduler.md

```diff
@@ -1,23 +1,21 @@
 ---
-title: "Working with the scheduler"
+title: "Queuing System Fundamentals"
```

Comment:
This title change introduces a mismatch with the first question

Reply:
Could it be renamed "Scheduling system fundamentals"? Keeps the gentler gist of the new title while retaining the word 'scheduler' to align with the first question.

Reply:
Rephrased the title & Q1 in f141db3.